package.json

```diff
@@ -25,13 +25,15 @@
     "@zxcvbn-ts/core": "^2.0.4",
     "classnames": "^2.2.5",
     "dompurify": "^2.3.3",
+    "justice-js-common-utils": "git+https://github.com/AccelByte/justice-js-common-utils.git#1.35.4",
```

Comment:
is this really gonna work (installing common-utils)? have you tried to install this version of ui-lib in AP and see that there's no error?

Reply:
yes, it's working fine, I tested it locally using `yalc`